db/migrate/20210522111400_create_incident_contacts.rb

```diff
@@ -0,0 +1,8 @@
+class CreateIncidentContacts < ActiveRecord::Migration[5.2]
+  def change
+    create_table :incident_contacts do |t|
+      t.references :incident, foreign_key: true
+      t.references :contact, foreign_key: true
```

Comment:
Just double checking, are we to also have title in our `incident_contacts` table?

Reply:
Great catch!